yarn.lock

```diff
@@ -13944,10 +13944,12 @@ use@^3.1.0:
   resolved "https://registry.yarnpkg.com/use/-/use-3.1.1.tgz#d50c8cac79a19fbc20f2911f56eb973f4e10070f"
   integrity sha512-cwESVXlO3url9YWlFW/TA9cshCEhtu7IKJ/p5soJ/gGpj7vbvFrAY/eIioQ6Dw23KjZhYgiIo8HOs1nQ2vr/oQ==
 
+
```

Comment:
These look like the result of a manual conflict resolution on this file. Since it's autogenerated, it's preferred to run `yarn` locally and commit + push to determine the content of this file.